webpack.dev.js

```diff
@@ -13,3 +13,9 @@ module.exports = merge(common, {
         host: '0.0.0.0',
     },
 })
+
+config.module.rules[3].use[1].options.modules = {
```

Comment:
This is certainly a bit brittle and will fail if, e.g. we change the order of the `rules` or `use` arrays in `webpack.common.js`. But the 'merge' command can probably not do this and in case it ever fails we will notice quickly? Another option would be turning the config into a function with arguments? Or maybe even change the `localIdentName` in the common config (also for production)?

Reply:
Ah ok I did not see this. I now adjusted the config accordingly.